config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  GOOGLE_CLIENT_ID:
+    "421850548520-50l00pri3hhnen7m1r40skhoaug0h0j9.apps.googleusercontent.com",
+  GOOGLE_CLIENT_SECRET: "ajgRPabjptefXlGcXHJQynbd",
+  CALLBACK_URL: "http://localhost:3000/auth/google/callback"
+};
```

Comment:
Wiem, że to trochę poza zakresem kursu, ale, takie parametry konfiguracyjne, najlepiej pobierać ze zmiennych środowiskowych w ten sposób `process.env.GOOGLE_CLIENT_ID` a nie wpisywać je na sztywno w kodzie, gdzie każdy mógłby podejrzeć nasze tajne klucze. 

Proponuje Ci skorzystać z biblioteki `dotenv` https://www.npmjs.com/package/dotenv

Korzystając z tej biblioteki tworzymy plik `.env` w projekcie który należy dodać do pliku `.gitignore` i w tym pliku definiujemy wymagane zmienne środowiskowe. Wówczas zmiana konfiguracji nie będzie wymagała ingerencji w kodzie.